### DISCUSSION TITLE
With multiple failing checks, the page doesn't say what's actually wrong

### DISCUSSION THREAD

**Discussion by spacecowboy** [WeblateOrg/weblate#5270] [Q&A] [answered]
Apologies if I'm just blind but where in this screen can I see what the failing check actually is??

This is a newly added project and I'm just getting to know the interface.

![Screenshot_2021-01-25 Feeder Android strings — English](https://user-images.githubusercontent.com/223655/105761679-b5166480-5f53-11eb-8610-39d734e37fdb.png)

**Comment by spacecowboy** [comment, marked as answer]
I might have answered by own question. The error is not in the english translation as one might think.

* Go to `Other languages`
* Scroll until you find an exclamation mark for a specific language - note that Russian tooltip says "Mismatched ellipsis"

So that's all fine. Would be good if it actually indicated which languages had issues instead of implying it found an issue in the original english..

**Reply by nijel** [reply]
The motivation behind this check is that it often indicates a problem in a source string, see https://docs.weblate.org/en/latest/user/checks.html#check-multiple-failures

Anyway listing the problematic checks directly is certainly a good idea, I've added that in 12101bd6dc40d826fb84a7d75e652b1be1d01e71. Thanks for your feedback!
